lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -125,29 +109,29 @@ def config_content
           end
 
           if config[:ssl_verify_mode]
-            client_rb << %Q{ssl_verify_mode :#{config[:ssl_verify_mode]}\n}
+            client_rb << %Q{ssl_verify_mode :#{chef_config[:ssl_verify_mode]}\n}
           end
 
-          if config[:bootstrap_proxy]
+          if chef_config[:bootstrap_proxy]
             client_rb << "\n"
-            client_rb << %Q{http_proxy        "#{config[:bootstrap_proxy]}"\n}
-            client_rb << %Q{https_proxy       "#{config[:bootstrap_proxy]}"\n}
-            client_rb << %Q{no_proxy          "#{config[:bootstrap_no_proxy]}"\n} if config[:bootstrap_no_proxy]
+            client_rb << %Q{http_proxy        "#{chef_config[:bootstrap_proxy]}"\n}
+            client_rb << %Q{https_proxy       "#{chef_config[:bootstrap_proxy]}"\n}
+            client_rb << %Q{no_proxy          "#{chef_config[:bootstrap_no_proxy]}"\n} if chef_config[:bootstrap_no_proxy]
           end
 
-          if config[:bootstrap_no_proxy]
-            client_rb << %Q{no_proxy       "#{config[:bootstrap_no_proxy]}"\n}
+          if chef_config[:bootstrap_no_proxy]
+            client_rb << %Q{no_proxy       "#{chef_config[:bootstrap_no_proxy]}"\n}
```

Comment:
same down here, pretty sure these need to be `config`

Reply:
Yes, Updated!